cpp/test/CMakeLists.txt

```diff
@@ -202,6 +202,9 @@ if(BUILD_PRIMS_TESTS)
     prims/seive.cu
     prims/sigmoid.cu
     prims/silhouetteScore.cu
+    prims/sparse/distance.cu
+    prims/sparse/knn.cu
+      prims/sparse/selection.cu
```

Comment:
Is this spaced strangely in the code or just a git diff error?

Reply:
CLion does this sometimes, for some reason. I've had to fix these a few times.